### PR TITLE
fix some format errors of Pod lifecycle

### PR DESCRIPTION
**Pull request by fancc**

### CODE REVIEW

**Comment by tengqm** [comment]
pls squash your commits

**Comment by sftim** [comment]
@fancc I hope it's OK to tweak the PR title for spelling:


/retitle fix some format errors of Pod lifecycle

File: content/zh/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 1]
```diff
@@ -256,8 +246,7 @@ the `Terminated` state.
 如果你使用 `kubectl` 来查询包含 `Terminated` 状态的容器的 Pod 时，你会看到
 容器进入此状态的原因、退出代码以及容器执行期间的起止时间。
 
-如果容器配置了 `preStop` 回调，则该回调会在容器进入 `Terminated`
-状态之前执行。
+PodSpec 中有一个 `restartPolicy` 字段，可能的值为 Always、OnFailure 和 Never。默认为 Always。`restartPolicy` 适用于 Pod 中的所有容器。`restartPolicy` 仅指通过同一节点上的 kubelet 重新启动容器。失败的容器由 kubelet 以五分钟为上限的指数退避延迟（10秒，20秒，40秒...）重新启动，并在成功执行十分钟后重置。如 [Pod 文档](/docs/user-guide/pods/#durability-of-pods-or-lack-thereof)中所述，一旦绑定到一个节点，Pod 将永远不会重新绑定到另一个节点。
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/pods/pod-lifecycle.md:249]: `[Pod 文档](/docs/user-guide/pods/#durability-of-pods-or-lack-thereof)`
This should point to zh version.

File: content/zh/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 2]
```diff
@@ -166,24 +166,18 @@ applies a policy for setting the `phase` of all Pods on the lost node to Failed.
 如果某节点死掉或者与集群中其他节点失联，Kubernetes
 会实施一种策略，将失去的节点上运行的所有 Pod 的 `phase` 设置为 `Failed`。
 
-<!--
-## Container states
+Pod 有一个 PodStatus 对象，其中包含一个 [PodCondition](/docs/resources-reference/v1.7/#podcondition-v1-core) 数组。PodCondition 数组的每个元素都有一个 `type` 字段和一个 `status` 字段。`type` 字段是字符串，可能的值有 PodScheduled、Ready、Initialized 和 Unschedulable。`status` 字段是一个字符串，可能的值有 True、False 和 Unknown。
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/pods/pod-lifecycle.md:169]: - Do we have English source for the translation above?
- PodCondition is not pointing to a valid location, it should be a valid, Chinese version page.

File: content/zh/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 3]
```diff
@@ -166,24 +166,18 @@ applies a policy for setting the `phase` of all Pods on the lost node to Failed.
...
 [container lifecycle hooks](/docs/concepts/containers/container-lifecycle-hooks/) to
 trigger events to run at certain points in a container's lifecycle.
 
-Once the {{< glossary_tooltip text="scheduler" term_id="kube-scheduler" >}}
-assigns a Pod to a Node, the kubelet starts creating containers for that Pod
-using a {{< glossary_tooltip text="container runtime" term_id="container-runtime" >}}.
-There are three possible container states: `Waiting`, `Running`, and `Terminated`.
--->
-## 容器状态  {#container-states}
+[探针](/docs/resources-reference/v1.7/#probe-v1-core)是由 [kubelet](/docs/admin/kubelet/) 对容器执行的定期诊断。要执行诊断，kubelet 调用由容器实现的 [Handler](https://godoc.org/k8s.io/kubernetes/pkg/api/v1#Handler)。有三种类型的处理程序：
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/pods/pod-lifecycle.md:176]: The first two links need to be revised.

**Pull request closed without merging**